packages/react/src/primitives/types/link.ts

```diff
@@ -16,6 +16,8 @@ export interface LinkOptions {
   isExternal?: boolean;
 
   /**
+   * @deprecated
+   * Please stop using this porp as it will be deprecated soon
```

Comment:
nit typo: 'prop'

Reply:
Props table be auto-generated at build time, but we may still want to register it in the code base?

Reply:
Same suggestion as above:

The Link component's `to` prop will soon be deprecated. Please see the Amplify UI documentation for using the Link component with routing libraries: https://ui.docs.amplify.aws/react/components/link#routing-libraries